pyth-sdk/src/price.rs

```diff
@@ -380,6 +672,18 @@ mod test {
         .unwrap()
     }
 
+    // arbitrary trait for quickcheck
+    impl Arbitrary for Price {
+        fn arbitrary(g: &mut Gen) -> Price {
+            Price {
+                price: i64::try_from(i32::arbitrary(g)).ok().unwrap(),
+                conf: 0,
```

Comment:
why are you using fixed values for these fields?

If you need restrictions, it may be better to explicitly pass the fields of the struct to the quickcheck macro below instead of implementing Arbitrary here.